streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -119,7 +120,7 @@
     private final String userEndPoint = "localhost:8080";
     private final String applicationId = "stream-partition-assignor-test";
 
-    private final TaskManager taskManager = EasyMock.createNiceMock(TaskManager.class);
+    private TaskManager taskManager;
```

Comment:
The story here is that (I think wrongly), the tests were using the task manager via the config map before, but not always initializing the mock. I guess a lot of the tests never accessed its methods before. I also guess that the mock just returns `null` for field accesses (specifically `adminClient`), but it doesn't validate the field access. Now that it's a method, it need the expectation of the call always, so we always need to initialize the mock now.

The changes I made in this class are related to this. Basically, we always initialize the taskManager mock before using it to configure the StreamsPartitionAssignor.